src/App.js

```diff
@@ -23,10 +23,10 @@ function App() {
       locallyLoadedMonsterNames.length === 0 ||
       locallyLoadedMonsterNames.length !== locallyLoadedMonsterDetails.length
     ) {
+      localStorage.clear()
```

Comment:
Just a question: why do you want to clear the localStorage each time App.js is rendered?

Reply:
This just doesn't happen  every time, only if not all the data is in the Localstorage yet. In that case the Localstorage gets wiped (to be sure i dont have fragments of data from before), and i fetch all the data again.

If the Localstorage already has all the data i require, he if condition above returns false and i load from Localstorage.

Reply:
Ahh okay :-) Thanks for the explanation!